src/ui/public/stringify/types/_numeral.js

```diff
@@ -4,46 +4,47 @@ import { FieldFormat } from 'ui/index_patterns/_field_format/field_format';
 
 const numeralInst = numeral();
 
-_.class(Numeral).inherits(FieldFormat);
-export function Numeral(params) {
-  Numeral.Super.call(this, params);
-}
-
-Numeral.prototype._convert = function (val) {
-  if (val === -Infinity) return '-∞';
-  if (val === +Infinity) return '+∞';
-  if (typeof val !== 'number') {
-    val = parseFloat(val);
+export class Numeral extends FieldFormat {
+  constructor(params) {
+    super(params);
   }
 
-  if (isNaN(val)) return '';
+  _convert(val) {
+    if (val === -Infinity) return '-∞';
+    if (val === +Infinity) return '+∞';
+    if (typeof val !== 'number') {
+      val = parseFloat(val);
+    }
 
-  return numeralInst.set(val).format(this.param('pattern'));
-};
+    if (isNaN(val)) return '';
 
+    return numeralInst.set(val).format(this.param('pattern'));
+  }
+}
 
 Numeral.factory = function (opts) {
-  _.class(Class).inherits(Numeral);
-  function Class(params, getConfig) {
-    Class.Super.call(this, params);
+  class Class extends Numeral {
+    constructor(params, getConfig) {
+      super(params);
 
-    this.getConfig = getConfig;
+      this.getConfig = getConfig;
+    }
+
+    getParamDefaults = function () {
```

Comment:
Since this is using class property syntax `getParamDefaults` will be redefined per instance (rather than attached to the prototype). Probably should use standard class method syntax.